src/components/WishlistPage/WishlistPage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import cx from 'classnames';
+import propTypes from 'prop-types';
+
+import WishlistItem from './WishlistItem';
+
+import './WishlistPage.scss';
+
+const CN = 'wishlist';
+
+const WishlistPage =({ wishlist, removeFromWishlist }) => {
+  const renderProduct = (item, removeFromWishlist) => {
+    return (
+      <WishlistItem key={item.id} item={item} removeFromWishlist={removeFromWishlist}/>
+    );
+  };
```

Comment:
🤔

Reply:
If there is only 1 component, then its simplier to use it in place instead of render function